.github/workflows/x-publish-npm.yml

```diff
@@ -0,0 +1,37 @@
+name: X Publish NPM packages
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+    secrets:
+      GH_ORG:
+        required: true
+      GH_TOKEN:
+        required: true
+      NPM_TOKEN:
+        required: true
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        repository:
+          - keycloak-nodejs-admin-client
+    steps:
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          check-latest: true
+
+      - name: Download tarball
+        run: gh release download ${{ inputs.tag }} --repo ${{ secrets.GH_ORG }}/${{ matrix.repository }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+
+      - name: Publish to NPM
+        run: npm publish ${{ matrix.repository }}.tgz --dry-run
```

Comment:
`--dry-run` would result in no actual publish happening? It might be appropriate to parameterize dry-run for the nightlies, to check it works for when you're doing a real release. However, this publish-npm workflow is called only for real releases as things are